logic/consensus/activationHeights.js

```diff
@@ -0,0 +1,7 @@
+const consensusActivationHeights = {
+  spaceship: 1_000_000,
```

Comment:
```suggestion
  spaceship: 100_000_000,
```

We already have 45M blocks :)